src/js/Templates/Shared/StaticIcon.js

```diff
@@ -13,7 +13,7 @@ export default class StaticIcon extends React.Component {
             var image = hex + value;
             path = "./" + image + ".svg"
             return (
-                <div className={this.props.class} >
+                <div className={this.props.class} style={{backgroundColor: this.props.fillColor}} >
```

Comment:
this background color should be applied in the control bar item class to the parent of the static icon, not in this class, as the bar item (soft button) is what is highlighted.

with the background, we should at least add `border-radius: 5px;` as well

Reply:
@iCollin Please take a look changes in commit 351e590
The `border-radius` added to `_control-bar.scss`. `backgroundColor` added as a `style` property of a `div` in a `ControlBarItem.js`